pkg/artifactory/resource_artifactory_user.go

```diff
@@ -10,9 +10,21 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
-	"github.com/jfrog/jfrog-client-go/artifactory/services"
 )
 
+type User struct {
+	Name                     string   `json:"name,omitempty" csv:"username,omitempty"`
```

Comment:
Are all these `omitempty` necessary? Why is there `csv` tag as well?

Reply:
omitempty is not necessary. In fact, In this specific attribute ie: profile_updatable it breaks the expected artifactory behavior.

Reply:
Let's remove all unnecessary `omitempty`.

Do we need `csv` tag?

Reply:
no need for csv tag. will remove them.

Reply:
csv??? Where did _that_ comes from?!

`omitempty` causes nothing but issues. Definitely remove them and set their default value to what RT would do; then it's super clear in the code the expected behavior